pkg/plugin/v3/scaffolds/init.go

```diff
@@ -39,7 +39,7 @@ const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
 	ControllerRuntimeVersion = "v0.6.2"
 	// ControllerToolsVersion is the kubernetes-sigs/controller-tools version to be used in the project
-	ControllerToolsVersion = "v0.3.0"
+	ControllerToolsVersion = "2f203e490e8af05e988fdac460151ff3a93f3ee7"
```

Comment:
We should use the release tag instead of the commit. See: v0.4.0

Reply:
I would like to. This PR should block on the next controller-tools release.

Reply:
H @estroz, I did not get. Is not the changes in the https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.4.0?

Reply:
No there is a feature needed in master that hasn't been released yet.

Reply:
So we are waiting for v0.4.1?